src/plugins/discover/public/embeddable/components/search_embeddable_grid_component.tsx

```diff
@@ -146,8 +160,8 @@ export function SearchEmbeddableGridComponent({
   const cellActionsMetadata = useAdditionalCellActions({
     dataSource,
     dataView,
-    query,
-    filters,
+    query: savedSearchQuery,
+    filters: savedSearchFilters,
```

Comment:
Can somebody confirm that it's what is expected? Or are we rather interested in the current Dashboard filters for cell actions?

Reply:
Yeah it's hard to say which behaviour is best for this. It's something worth discussing at least.